src/components/BoardConfiguration.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="board-config">
+    <div class="div-group">
+        <label for="cols" class="my-label">Columns Number:</label>
+        <input type="number" id="cols" name="cols" class="my-text" min="1" max="50" v-model="ColsConfig">
+    </div>
+    <div class="div-group">
+      <label for="rows" class="my-label">Rows Number:</label>
+      <input type="number" id="rows" name="rows" class="my-text" min="1" max="50" v-model="RowsConfig">
+    </div>
+    <div class="div-group">
+      <input type="button" id="button" class="my-button" name="button" value="Create Board" @click="createBoard()">
+    </div> 
+  </div>
+</template>
+
+<script>
+import {BoardBus} from '@/services/board-bus';
+export default {
+  name: 'board-config',
+  data() {
+    return {
+      RowsConfig: 1,
+      ColsConfig: 1
+    }
+  },  
+  methods: {
+    createBoard() {
```

Comment:
you could make it simpler with early returns:
```javascript
if(!(this.columns <= 50 && this.columns > 0) || !(this.rows <= 50 && this.rows > 0)) {
 return alert("The values of the number of rows or columns couldn't be more than 60 or less than 1.");
}

BoardBus.$emit('values-config', {
  cols: this.columns,
  rows: this.rows,
});

BoardBus.$emit('token-config', this.randomKey());
```

Reply:
The validation and the Bus were changed in the last commits.